public/index.html

```diff
@@ -41,3 +44,6 @@
     -->
   </body>
 </html>
+<!-- Compiled and minified JavaScript -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```

Comment:
Let's pick one of materialize and bootstrap

Reply:
I will delete this materialize.